source/main/index.js

```diff
@@ -96,6 +98,30 @@ const onAppReady = async () => {
   const isInSafeMode = includes(process.argv.slice(1), '--safe-mode');
 
   mainWindow = createMainWindow(isInSafeMode);
+
+  const onCheckDiskSpace = ({ notEnoughSpace }: CheckDiskSpaceResponse) => {
+    if (cardanoNode) {
+      if (notEnoughSpace) {
+        try {
+          cardanoNode.stop();
+        } catch (e) {} // eslint-disable-line
+      } else if (
+        cardanoNode.state !== CardanoNodeStates.STARTING &&
+        cardanoNode.state !== CardanoNodeStates.RUNNING
+      ) {
+        restartCardanoNode(cardanoNode);
```

Comment:
Why not use `node.restart()` directly here?

Reply:
That's odd, I tried before and got some errors. Must have done something wrong. Changed, now.